ssl/statem/statem_clnt.c

```diff
@@ -3057,9 +3057,9 @@ static int tls_construct_cke_dhe(SSL *s, WPACKET *pkt)
 {
 #ifndef OPENSSL_NO_DH
     DH *dh_clnt = NULL;
-    const BIGNUM *pub_key;
     EVP_PKEY *ckey = NULL, *skey = NULL;
     unsigned char *keybytes = NULL;
+    int np;
```

Comment:
I'd consider a different name for this variable, maybe `prime_len` -- I'm not actually sure what "np" is supposed to indicate or stand for.